docs/components/collapse.md

```diff
@@ -143,6 +143,128 @@ To set the default state (collapsed or expanded) without binding to v-model you
   I am expanded by default
 </KCollapse>
 ```
+### type
+
+Values: `tags` or `default`. Use this optional prop to specific render the tags(`KBadge`) content.  The trigger icon will automatically appear when the content width is longer than the parent container. If type is setted to tags, only tags related props: `tags`, `tagsAppearance`, and `noCollapse` will take effect.
```

Comment:
question: I don't understand what this means:
```
Values: `tags` or `default`
```

What is default? What are tags, an array?